lib/cisco_node_utils/cmd_ref/vxlan_vtep_vni.yaml

```diff
@@ -18,7 +18,7 @@ all_vnis:
 ingress_replication:
   kind: string
   config_get_token_append: '/^ingress-replication protocol (\S+)$/'
-  config_set_append: '<state> ingress-replication protocol <protocol>'
+  config_set_append: '<state> ingress-replication protocol <protocol> ; end'
```

Comment:
Removed 'end' for ingress_replication

On Wed, Jan 27, 2016 at 9:58 AM, Mike Wiebe <notifications@github.com>
wrote:

> In lib/cisco_node_utils/cmd_ref/vxlan_vtep_vni.yaml
> <https://github.com/cisco/cisco-network-node-utils/pull/190#discussion_r51022272>
> :
>
> > @@ -18,7 +18,7 @@ all_vnis:
> >  ingress_replication:
> >    kind: string
> >    config_get_token_append: '/^ingress-replication protocol (\S+)$/'
> > -  config_set_append: '<state> ingress-replication protocol <protocol>'
> > +  config_set_append: '<state> ingress-replication protocol <protocol> ; end'
>
> The end here really should not be needed. Did it fail without it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cisco/cisco-network-node-utils/pull/190/files#r51022272>
> .
>